<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginFrameClass</class>
 <widget class="QMainWindow" name="PluginFrameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PluginFrame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>598</width>
         <height>319</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_image">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_config"/>
   </widget>
   <widget class="QMenu" name="menu_Plugin">
    <property name="title">
     <string>插件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_large"/>
    <addaction name="action_small"/>
   </widget>
   <addaction name="menu_Menu"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Plugin"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_close"/>
   <addaction name="separator"/>
   <addaction name="action_small"/>
   <addaction name="action_large"/>
   <addaction name="separator"/>
   <addaction name="action_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_new">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/新建.png</normaloff>:/Resources/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/打开.bmp</normaloff>:/Resources/打开.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/关闭.png</normaloff>:/Resources/关闭.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_config">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/设置.png</normaloff>:/Resources/设置.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_large">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/放大.png</normaloff>:/Resources/放大.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_small">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/缩小.png</normaloff>:/Resources/缩小.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="PluginFrame.qrc">
     <normaloff>:/Resources/帮助.png</normaloff>:/Resources/帮助.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>查看帮助</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PluginFrame.qrc"/>
 </resources>
 <connections/>
</ui>
